### PR TITLE
bug: fix enrollments export

### PR DESCRIPTION
**Pull request by alves-luis**
When exporting the enrollments, a 500 error would be thrown. This fixes the ability to export current enrollments.

### CODE REVIEW

**Comment by fntneves** [comment]
In what scenario the 500 was thrown?

Provide more details, please.

**Comment by alves-luis** [comment]
With 0 enrollments, pressing the _Export enrollments_ button throws this error:

```
Symfony\Component\Debug\Exception\FatalThrowableError
Typed property App\Exports\EnrollmentsExport::$enrollments must be an instance of Illuminate\Support\Collection, null used
```

File: app/Exports/EnrollmentsExport.php [hunk 1]
```diff
@@ -8,9 +8,12 @@
 use Maatwebsite\Excel\Concerns\WithMapping;
 use Maatwebsite\Excel\Concerns\WithHeadings;
 use Maatwebsite\Excel\Concerns\FromCollection;
+use Maatwebsite\Excel\Concerns\Exportable;
 
 class EnrollmentsExport implements FromCollection, WithMapping, WithHeadings
 {
+    use Exportable;
```
Comment (fntneves) [inline app/Exports/EnrollmentsExport.php]: Is this mandatory?

Comment (alves-luis) [reply app/Exports/EnrollmentsExport.php]: In _Http/Controllers/Admin/EnrollmentController.php_, the export is done by assuming that the EnrollmentsExport has this trait, as in:

`(new EnrollmentsExport())->download`

To maintain that structure, this change is mandatory. However, we could not include the Exportable trait if we instead used the _Excel::download_ facade, like this (in the controller):

`Excel::download(new EnrollmentsExport, 'enrollments.csv', ...)`

For reference: [Exporting collections](https://docs.laravel-excel.com/3.0/exports/collection.html)

**Pull request merged**